File: tasks/translation/output/Qwen/Qwen2.5-32B-Instruct-AWQ/codenet/cot/Java/Go/s826491904_comments_removed.go
package main

import (
	"fmt"
)

func main() {
	solve()
}

func solve() {
	var N int
	fmt.Scanf("%d", &N)
	var max, min int
	fmt.Scanf("%d", &max)
	min = max
	for i := 1; i < N; i++ {
		var in int
		fmt.Scanf("%d", &in)
		if in > max {
			max = in
		}
		if in < min {
			min = in
		}
	}
	fmt.Println(max - min) // End of Code

File: tasks/translation/output/Qwen/Qwen2.5-32B-Instruct-AWQ/codenet/cot/Java/C/s826491904_comments_removed.c
#include <stdio.h>

void solve() {
    int N, max, min, in;
    scanf("%d", &N);
    scanf("%d", &max);
    min = max;
    for (int i = 1; i < N; i++) {
        scanf("%d", &in);
        if (in > max) max = in;
        if (in < min) min = in;
    }
    printf("%d\n", max - min);
}

int main() {
    solve();
    return 0;
} // End of Code
